src/content/data-streams/index.mdx

```diff
@@ -68,13 +68,13 @@ One example of how to use Data Streams is in a decentralized exchange. An exampl
 1. A user initiates a trade by confirming an `initiateTrade` transaction in their wallet.
 1. The onchain contract for the decentralized exchange responds by emitting a Log Trigger event.
 1. The Chainlink Automation upkeep monitors the contract for the event. When Automation detects the event, it runs the `checkLog` function specified in the upkeep contract. The upkeep is defined by the decentralized exchange.
-1. The `checkLog` function emits an event called `StreamsLookup`.
-1. Data Streams detects the `StreamsLookup` event and returns the requested report in the `checkCallback` function for Chainlink Automation.
+1. The `checkLog` function uses a `revert` with a custom error called `StreamsLookup`. This approach, aligning with [EIP-3668](https://eips.ethereum.org/EIPS/eip-3668#use-of-revert-to-convey-call-information), conveys necessary information through the data in the `revert` custom error.
```

Comment:
```suggestion
1. The `checkLog` function uses a `revert` with a custom error called `StreamsLookup`. This approach aligns with [EIP-3668](https://eips.ethereum.org/EIPS/eip-3668#use-of-revert-to-convey-call-information) and conveys the required information through the data in the `revert` custom error.
```

Avoiding gerunds makes sentences easier to read and usually easier to translate.